ssv/spectest/tests/runner/consensus/valid_decided_10_operators.go

```diff
@@ -20,7 +20,7 @@ func ValidDecided10Operators() tests.SpecTest {
 				Runner:                  testingutils.SyncCommitteeContributionRunner(ks),
 				Duty:                    &testingutils.TestingSyncCommitteeContributionDuty,
 				Messages:                testingutils.SSVDecidingMsgsV(testingutils.TestSyncCommitteeContributionConsensusData, ks, types.BNRoleSyncCommitteeContribution),
-				PostDutyRunnerStateRoot: "ec47a2194bab81891bd581eaf427af211b53e13d1f77a6703eeb56a693caa878",
+				PostDutyRunnerStateRoot: "a991b6470a8c7a55f4ce89aea91925c2d80a9a8c4258545cc2fb17cabc388719",
```

Comment:
why is this changed? don't we take post root from json?

Reply:
ditto

Reply:
So currently there are 2 issues:
1. We have that force backwards compatability check
2. Currently we generate `tests.json` before we override the roots... so we must have the correct root in place